articles/customizing-django-admin/index.md

```diff
@@ -0,0 +1,245 @@
+### Customizing Django Admin
+Django is a Web framework for perfectionists with deadlines. That is why it comes with its admin site. Django gives programmers the power of customizing this admin site to suit the requirements of the project or the needs of the client.
+
+### Prerequisites
+Before we begin, you should have the following:
+ - Python3 installed on your machine.
+ - Django installed on a virtual environment on your machine.
+ - Some knowledge of Python 3 and Django.
+
+### Step 1 – Creating our Django app
+Open the command line and execute the following command to create the project.
+```bash 
+$ django-admin startproject Custom
+```
+Now let's change to the created directory `Custom`.
+```bash
+$ cd Custom
+```
+In `Custom`, let's go ahead and create our app called "Bookstore".
+```bash
+$ django-admin startapp Bookstore
+```
+Then, add Bookstore to the installed apps in `settings.py`.
+```py
+INSTALLED_APPS  = [
+  # ...
+  'Bookstore' # new
+]
+```
+Next, use the following command to create a virtual environment in the project folder level.
+```bash
+$ py -m venv .venv
+```
+Then, activate it using the following command.
+
+```bash
+$ .venv\Scripts\activate.bat
+```
+
+Now we can install Django in the virtual environment.
+
+```bash
+$ pip install django
+$ pip install django
+```
```

Comment:
You have repeated this command twice. Please remove one.

Reply:
fixed